docs/examples/pyangbind/index.md

```diff
@@ -0,0 +1,145 @@
+
+PyangBind is a pyang plugin.
+
+## About Pyang
+
+pyang is a python program.
+
+We can use it to:
+
+- Validate YANG modules against YANG RFCs
+- Convert YANG modules into equivalent YIN module (XML)
+- Generate a tree representation of YANG models for quick visualization
+
+## About PyangBind
+
+It generates Python classes from a YANG module.
+
+It converts YANG module into a Python module, such that Python can be used to generate data which conforms with the data model defined in YANG.
+
+## Install Pyang and Pyangbind
+
+```shell
+pip install pyang
+pip install pyangbind
+```
+
+```shell
+pip3 freeze | grep pyang
+```
+
+## We need yang modules
+
+### Create a directory
+
+```shell
+mkdir yang_modules
+```
+
+### Clone the OpenConfig repository
+
+
+```shell
+git clone https://github.com/openconfig/public.git
+```
+```
+ls public
+```
```

Comment:
if there's no output, you can probably pull this.

Reply:
There is an output, but there is no benefit to show it that's why I am not including it. I just wanted to share the command to check.